drivers/quota/projectquota.go

```diff
@@ -141,6 +163,12 @@ func NewControl(basePath string) (*Control, error) {
 		return nil, err
 	}
 
+	if q.nextProjectID == 1 {
```

Comment:
What does this mean? The project ID is computed by `getProjectID` +` findNextProjectID` + some increments. Under which conditions would it end up 1?

This might well be correct, but please comment this to clearly document the intent and logic instead of just using a magic `1`.

Also, isn’t this too late, if `setProjectQuota` was already called on `minProjectID`?

Reply:
It will only be 1 if no quota has ever been set.  setProjectQuota above is checking to see if quota is supported.

Reply:
Then please make `getProjectID` document that it returns 0 (without error) on a non-quota directory, and add a comment to this condition as well.

Reply:
… the `setProjectQuota` call above _is_ well-documented, my bad for not noticing.

Reply:
I have modified the code to simplify.